evap/development/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
```

Comment:
If codecov complains that the lines here and in views.py are uncovered: I'd vote in favor of ignoring these files, I consider them to be expected to be untested. https://docs.codecov.io/docs/ignoring-paths